drivers/bluetooth/adafruit_ble.cpp

```diff
@@ -673,6 +679,32 @@ void adafruit_ble_send_mouse_move(int8_t x, int8_t y, int8_t scroll, int8_t pan,
 
 uint32_t adafruit_ble_read_battery_voltage(void) { return state.vbat; }
 
+// https://learn.adafruit.com/adafruit-feather-32u4-bluefruit-le/power-management#measuring-battery-2177042-9
```

Comment:
In consideration of this page, how about to provide it as the default behavior, but also allows user to override?